gallery/how-to/managing-profile/managing-account.md

```diff
@@ -6,82 +6,94 @@ title:  PowerShell Gallery Account Settings
 ---
 # PowerShell Gallery Account Settings
 
-Your PowerShell Gallery account is a publicly visible name that is linked to either an email account and a login that is either a Microsoft ID (such as user@hotmail.com, or user@outlook.com), or an Azure Active Directory (AAD) account. 
-Account Settings provides you:
+Your PowerShell Gallery account is a publicly visible name that is linked to an identity. That
+identity is either a Microsoft ID, like `user@hotmail.com` or `user@outlook.com`, or an Azure
+Active Directory (AAD) account.
 
-* Control over the email account associated with your PowerShell Gallery account;
-* Control over certain email notifications you can receive from the PowerShell Gallery;
-* Ability to change the login account, which is the Microsoft ID or AAD account linked to your PowerShell Gallery account;
-* How to set the graphic that is displayed beside your PowerShell Gallery account name
+The PowerShell Gallery provides the following account settings:
 
+- The email account associated with your PowerShell Gallery account
+- Options for email notifications sent from the PowerShell Gallery
+- The user account associated with your PowerShell Gallery account
+- The picture associated with your PowerShell Gallery account
 
+## Email address
 
-## Email Address
-
-The email address is where the PowerShell Gallery will send email notifications.
-This email address is never provided to other users directly by the PowerShell Gallery. 
-You may use any email account you have access to, and it does not have to match the login account. 
+The email address is the destination for PowerShell Gallery notifications. It does not have to
+match the login account. You may use any email account you have access to. Your email address is
```

Comment:
Missing one piece: "The email address is never given to other users directly by the PowerShell Gallery."

Reply:
fixed